src/core/adapters/officialOnyxiaApiClient.ts

```diff
@@ -139,6 +139,8 @@ export function createOfficialOnyxiaApiClient(params: {
                             expose: {
                                 domain: string;
                                 ingressClassName: string;
+                                ingress: boolean;
+                                route: boolean;
```

Comment:
I think thoses properties are both potentiallyundefined. 
```suggestion
                                ingress?: boolean;
                                route? : boolean;
```

Reply:
The same as `ingressClassName` and `domain` when a chart exposes nothing. Not sure if its true, I followed the existing code.